The insert statement is abnormal."ERROR:  Cannot invoke "org.antlr.v4.runtime.tree.ParseTree.accept(org.antlr.v4.runtime.tree.ParseTreeVisitor)" "

### Which version of ShardingSphere did you use?
we find java version: java17, full_version=17.0.1
ShardingSphere-5.2.0

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-Proxy

### Expected behavior
The insert statement is normal.

### Actual behavior
The insert statement is abnormal.

### Reason analyze (If you can)

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
```
drop table customer;							 
CREATE TABLE CUSTOMER ( C_CUSTKEY INTEGER NOT NULL,
C_NAME VARCHAR(25) NOT NULL,
C_ADDRESS VARCHAR(40) NOT NULL,
C_NATIONKEY INTEGER NOT NULL,
C_PHONE CHAR(15) NOT NULL,
C_ACCTBAL DECIMAL(15,2)   NOT NULL,
C_MKTSEGMENT CHAR(10) NOT NULL,
C_COMMENT VARCHAR(117) NOT NULL);

INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (107451, 'Customer#000107451', 'jZiY8FZz1pESpfE0ZvicC4f9evS1lrF', 24, '34-474-838-7669', -709.94, 'FURNITURE ', 'ans. furiously final theodolites are carefully. carefully final theodolites alongside of the');   --abnormal

INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c'); --normal
```
**background log**
```
[ERROR] 2022-09-24 10:10:28.595 [epollEventLoopGroup-3-7] o.a.s.p.f.c.CommandExecutorTask - Exception occur:
java.lang.NullPointerException: Cannot invoke "org.antlr.v4.runtime.tree.ParseTree.accept(org.antlr.v4.runtime.tree.ParseTreeVisitor)" because "tree" is null
        at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createCommonBinaryOperationSegment(OpenGaussStatementSQLVisitor.java:348)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitAExpr(OpenGaussStatementSQLVisitor.java:303)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitAExpr(OpenGaussStatementSQLVisitor.java:194)
        at org.apache.shardingsphere.sql.parser.autogen.OpenGaussStatementParser$AExprContext.accept(OpenGaussStatementParser.java:14788)
        at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:770)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:767)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:767)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:758)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsertRest(OpenGaussStatementSQLVisitor.java:696)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsertRest(OpenGaussStatementSQLVisitor.java:194)
        at org.apache.shardingsphere.sql.parser.autogen.OpenGaussStatementParser$InsertRestContext.accept(OpenGaussStatementParser.java:3231)
        at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsert(OpenGaussStatementSQLVisitor.java:632)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsert(OpenGaussStatementSQLVisitor.java:194)
        at org.apache.shardingsphere.sql.parser.autogen.OpenGaussStatementParser$InsertContext.accept(OpenGaussStatementParser.java:3089)
        at org.apache.shardingsphere.sql.parser.api.SQLVisitorEngine.visit(SQLVisitorEngine.java:54)
        at org.apache.shardingsphere.infra.parser.sql.SQLStatementParserExecutor.parse(SQLStatementParserExecutor.java:48)
        at org.apache.shardingsphere.infra.parser.sql.SQLStatementParserEngine.parse(SQLStatementParserEngine.java:47)
        at org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine.parse(ShardingSphereSQLParserEngine.java:58)
        at org.apache.shardingsphere.proxy.backend.handler.ProxyBackendHandlerFactory.newInstance(ProxyBackendHandlerFactory.java:86)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.query.simple.OpenGaussComQueryExecutor.<init>(OpenGaussComQueryExecutor.java:72)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.OpenGaussCommandExecutorFactory.getCommandExecutor(OpenGaussCommandExecutorFactory.java:114)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.OpenGaussCommandExecutorFactory.newInstance(OpenGaussCommandExecutorFactory.java:76)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.OpenGaussCommandExecuteEngine.getCommandExecutor(OpenGaussCommandExecuteEngine.java:62)
        at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.executeCommand(CommandExecutorTask.java:108)
        at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.run(CommandExecutorTask.java:77)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:833)
```



### Example codes for reproduce this issue (such as a github link).
```
rules:
- !SHARDING
  tables:
    lineitem:
      actualDataNodes: ds_${0..1}.lineitem_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: l_partkey
          shardingAlgorithmName: lineitem-database-inline
      tableStrategy:
        standard:
          shardingColumn: l_orderkey
          shardingAlgorithmName: lineitem-table-inline
    customer:
      actualDataNodes: ds_${0..1}.customer_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: c_custkey
          shardingAlgorithmName: customer-database-inline
      tableStrategy:
        standard:
          shardingColumn: c_nationkey
          shardingAlgorithmName: customer-table-inline
    nation:
      actualDataNodes: ds_${0..1}.nation_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: n_nationkey
          shardingAlgorithmName: nation-database-inline
      tableStrategy:
        standard:
          shardingColumn: n_regionkey
          shardingAlgorithmName: nation-table-inline
    orders:
      actualDataNodes: ds_${0..1}.orders_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: o_orderkey
          shardingAlgorithmName: orders-database-inline
      tableStrategy:
        standard:
          shardingColumn: o_custkey
          shardingAlgorithmName: orders-table-inline
    part:
      actualDataNodes: ds_${0..1}.part_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: p_partkey
          shardingAlgorithmName: part-database-inline
      tableStrategy:
        standard:
          shardingColumn: p_size
          shardingAlgorithmName: part-table-inline
    partsupp:
      actualDataNodes: ds_${0..1}.partsupp_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: ps_partkey
          shardingAlgorithmName: partsupp-database-inline
      tableStrategy:
        standard:
          shardingColumn: ps_suppkey
          shardingAlgorithmName: partsupp-table-inline
    region:
      actualDataNodes: ds_${0..1}.region
      databaseStrategy:
        standard:
          shardingColumn: r_regionkey
          shardingAlgorithmName: region-database-inline
    supplier:
      actualDataNodes: ds_${0..1}.supplier_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: s_suppkey
          shardingAlgorithmName: supplier-database-inline
      tableStrategy:
        standard:
          shardingColumn: s_nationkey
          shardingAlgorithmName: supplier-table-inline
  defaultDatabaseStrategy:
    none:
  defaultTableStrategy:
    none:

  shardingAlgorithms:
    lineitem-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${l_partkey % 2}
    lineitem-table-inline:
      type: INLINE
      props:
        algorithm-expression: lineitem_${l_orderkey % 2}
    customer-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${c_custkey % 2}
    customer-table-inline:
      type: INLINE
      props:
        algorithm-expression: customer_${c_nationkey % 2}
    nation-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${n_nationkey % 2}
    nation-table-inline:
      type: INLINE
      props:
        algorithm-expression: nation_${n_regionkey % 2}
    orders-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${o_orderkey % 2}
    orders-table-inline:
      type: INLINE
      props:
        algorithm-expression: orders_${o_custkey % 2}
    part-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${p_partkey % 2}
    part-table-inline:
      type: INLINE
      props:
        algorithm-expression: part_${p_size % 2}
    partsupp-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${ps_partkey % 2}
    partsupp-table-inline:
      type: INLINE
      props:
        algorithm-expression: partsupp_${ps_suppkey % 2}
    region-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${r_regionkey % 2}
    supplier-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${s_suppkey % 2}
    supplier-table-inline:
      type: INLINE
      props:
        algorithm-expression: supplier_${s_nationkey % 2}
schemaName: tpch_db_2
```

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/BaseRule.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/BaseRule.g4
@@ -76,7 +76,7 @@ stringLiterals
     ;
     
 numberLiterals
-    : NUMBER_
+    : (PLUS_ | MINUS_)? NUMBER_
     ;
     
 temporalLiterals
@@ -1234,11 +1234,6 @@ defaultEncryption
     
 defaultCharset
     : DEFAULT? charset EQ_? charsetName
-    ;
-    
-signedLiteral
-    : literals
-    | (PLUS_ | MINUS_) numberLiterals
     ;
     
 now

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DDLStatement.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DDLStatement.g4
@@ -96,7 +96,7 @@ alterListItem
     | DROP (COLUMN? columnInternalRef=identifier restrict? | FOREIGN KEY columnInternalRef=identifier | PRIMARY KEY | keyOrIndex indexName | CHECK identifier | CONSTRAINT identifier)  # alterTableDrop
     | DISABLE KEYS  # disableKeys
     | ENABLE KEYS   # enableKeys
-    | ALTER COLUMN? columnInternalRef=identifier (SET DEFAULT (LP_ expr RP_| signedLiteral)| SET visibility | DROP DEFAULT) # alterColumn
+    | ALTER COLUMN? columnInternalRef=identifier (SET DEFAULT (LP_ expr RP_| literals)| SET visibility | DROP DEFAULT) # alterColumn
     | ALTER INDEX indexName visibility  # alterIndex
     | ALTER CHECK constraintName constraintEnforcement  # alterCheck
     | ALTER CONSTRAINT constraintName constraintEnforcement # alterConstraint
@@ -435,7 +435,7 @@ fieldDefinition
 columnAttribute
     : NOT? NULL
     | NOT SECONDARY
-    | value = DEFAULT (signedLiteral | now | LP_ expr RP_)
+    | value = DEFAULT (literals | now | LP_ expr RP_)
     | value = ON UPDATE now
     | value = AUTO_INCREMENT
     | value = SERIAL DEFAULT VALUE

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/antlr4/imports/opengauss/BaseRule.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/antlr4/imports/opengauss/BaseRule.g4
@@ -938,13 +938,17 @@ funcName
     ;
 
 aexprConst
-    : NUMBER_
+    : numberConst
     | STRING_
     | funcName STRING_
     | funcName LP_ funcArgList sortClause? RP_ STRING_
     | TRUE
     | FALSE
     | NULL
+    ;
+
+numberConst
+    : (PLUS_ | MINUS_)? NUMBER_
     ;
 
 qualifiedName

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/java/org/apache/shardingsphere/sql/parser/opengauss/visitor/statement/impl/OpenGaussStatementSQLVisitor.java
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/java/org/apache/shardingsphere/sql/parser/opengauss/visitor/statement/impl/OpenGaussStatementSQLVisitor.java
@@ -298,16 +298,19 @@ public abstract class OpenGaussStatementSQLVisitor extends OpenGaussStatementBas
         if (null != ctx.patternMatchingOperator()) {
             return createPatternMatchingOperationSegment(ctx);
         }
-        Optional<String> commonBinaryOperator = findCommonBinaryOperator(ctx);
-        if (commonBinaryOperator.isPresent()) {
-            return createCommonBinaryOperationSegment(ctx, commonBinaryOperator.get());
+        Optional<String> binaryOperator = findBinaryOperator(ctx);
+        if (binaryOperator.isPresent()) {
+            return createBinaryOperationSegment(ctx, binaryOperator.get());
         }
         super.visitAExpr(ctx);
         String text = ctx.start.getInputStream().getText(new Interval(ctx.start.getStartIndex(), ctx.stop.getStopIndex()));
         return new CommonExpressionSegment(ctx.getStart().getStartIndex(), ctx.getStop().getStopIndex(), text);
     }
     
-    private Optional<String> findCommonBinaryOperator(final AExprContext ctx) {
+    private Optional<String> findBinaryOperator(final AExprContext ctx) {
+        if (1 == ctx.aExpr().size()) {
+            return Optional.empty();
+        }
         if (null != ctx.comparisonOperator()) {
             return Optional.of(ctx.comparisonOperator().getText());
         }
@@ -343,7 +346,7 @@ public abstract class OpenGaussStatementSQLVisitor extends OpenGaussStatementBas
         return new BinaryOperationExpression(ctx.start.getStartIndex(), ctx.stop.getStopIndex(), left, right, operator, text);
     }
     
-    private BinaryOperationExpression createCommonBinaryOperationSegment(final AExprContext ctx, final String operator) {
+    private BinaryOperationExpression createBinaryOperationSegment(final AExprContext ctx, final String operator) {
         ExpressionSegment left = (ExpressionSegment) visit(ctx.aExpr(0));
         ExpressionSegment right = (ExpressionSegment) visit(ctx.aExpr(1));
         String text = ctx.start.getInputStream().getText(new Interval(ctx.start.getStartIndex(), ctx.stop.getStopIndex()));
@@ -438,8 +441,8 @@ public abstract class OpenGaussStatementSQLVisitor extends OpenGaussStatementBas
     
     @Override
     public ASTNode visitAexprConst(final AexprConstContext ctx) {
-        if (null != ctx.NUMBER_()) {
-            return new NumberLiteralValue(ctx.NUMBER_().getText());
+        if (null != ctx.numberConst()) {
+            return new NumberLiteralValue(ctx.numberConst().getText());
         }
         if (null != ctx.STRING_()) {
             return new StringLiteralValue(ctx.STRING_().getText());

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4
@@ -933,7 +933,7 @@ funcName
     ;
 
 aexprConst
-    : NUMBER_
+    : numberConst
     | STRING_
     | BEGIN_DOLLAR_STRING_CONSTANT DOLLAR_TEXT* END_DOLLAR_STRING_CONSTANT
     | funcName STRING_
@@ -941,6 +941,10 @@ aexprConst
     | TRUE
     | FALSE
     | NULL
+    ;
+
+numberConst
+    : (PLUS_ | MINUS_)? NUMBER_
     ;
 
 qualifiedName

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/java/org/apache/shardingsphere/sql/parser/postgresql/visitor/statement/impl/PostgreSQLStatementSQLVisitor.java
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/java/org/apache/shardingsphere/sql/parser/postgresql/visitor/statement/impl/PostgreSQLStatementSQLVisitor.java
@@ -298,16 +298,19 @@ public abstract class PostgreSQLStatementSQLVisitor extends PostgreSQLStatementP
         if (null != ctx.patternMatchingOperator()) {
             return createPatternMatchingOperationSegment(ctx);
         }
-        Optional<String> commonBinaryOperator = findCommonBinaryOperator(ctx);
-        if (commonBinaryOperator.isPresent()) {
-            return createCommonBinaryOperationSegment(ctx, commonBinaryOperator.get());
+        Optional<String> binaryOperator = findBinaryOperator(ctx);
+        if (binaryOperator.isPresent()) {
+            return createBinaryOperationSegment(ctx, binaryOperator.get());
         }
         super.visitAExpr(ctx);
         String text = ctx.start.getInputStream().getText(new Interval(ctx.start.getStartIndex(), ctx.stop.getStopIndex()));
         return new CommonExpressionSegment(ctx.getStart().getStartIndex(), ctx.getStop().getStopIndex(), text);
     }
     
-    private Optional<String> findCommonBinaryOperator(final AExprContext ctx) {
+    private Optional<String> findBinaryOperator(final AExprContext ctx) {
+        if (1 == ctx.aExpr().size()) {
+            return Optional.empty();
+        }
         if (null != ctx.comparisonOperator()) {
             return Optional.of(ctx.comparisonOperator().getText());
         }
@@ -343,7 +346,7 @@ public abstract class PostgreSQLStatementSQLVisitor extends PostgreSQLStatementP
         return new BinaryOperationExpression(ctx.start.getStartIndex(), ctx.stop.getStopIndex(), left, right, operator, text);
     }
     
-    private BinaryOperationExpression createCommonBinaryOperationSegment(final AExprContext ctx, final String operator) {
+    private BinaryOperationExpression createBinaryOperationSegment(final AExprContext ctx, final String operator) {
         ExpressionSegment left = (ExpressionSegment) visit(ctx.aExpr(0));
         ExpressionSegment right = (ExpressionSegment) visit(ctx.aExpr(1));
         String text = ctx.start.getInputStream().getText(new Interval(ctx.start.getStartIndex(), ctx.stop.getStopIndex()));
@@ -438,8 +441,8 @@ public abstract class PostgreSQLStatementSQLVisitor extends PostgreSQLStatementP
     
     @Override
     public ASTNode visitAexprConst(final AexprConstContext ctx) {
-        if (null != ctx.NUMBER_()) {
-            return new NumberLiteralValue(ctx.NUMBER_().getText());
+        if (null != ctx.numberConst()) {
+            return new NumberLiteralValue(ctx.numberConst().getText());
         }
         if (null != ctx.STRING_()) {
             return new StringLiteralValue(ctx.STRING_().getText());

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sql92/src/main/antlr4/imports/sql92/BaseRule.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sql92/src/main/antlr4/imports/sql92/BaseRule.g4
@@ -38,8 +38,8 @@ stringLiterals
     ;
 
 numberLiterals
-   : MINUS_? NUMBER_
-   ;
+    : (PLUS_ | MINUS_)? NUMBER_
+    ;
 
 dateTimeLiterals
     : (DATE | TIME | TIMESTAMP) STRING_

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/BaseRule.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/BaseRule.g4
@@ -38,8 +38,8 @@ stringLiterals
     ;
 
 numberLiterals
-   : MINUS_? NUMBER_
-   ;
+    : (PLUS_ | MINUS_)? NUMBER_
+    ;
 
 dateTimeLiterals
     : (DATE | TIME | TIMESTAMP) STRING_

--- a/shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/dml/insert.xml
+++ b/shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/dml/insert.xml
@@ -100,7 +100,7 @@
         </values>
     </insert>
 
-    <insert sql-case-id="insert_with_columnname_uuid" parameters="1, 1">
+    <insert sql-case-id="insert_with_uuid_column" parameters="1, 1">
         <table name="t_order" start-index="12" stop-index="18" />
         <columns start-index="20" stop-index="28">
             <column name="id" start-index="21" stop-index="22" />
@@ -2085,5 +2085,30 @@
                 </assignment-value>
             </assignment>
         </on-duplicate-key-columns>
+    </insert>
+
+    <insert sql-case-id="insert_with_negative_value" parameters="1, -1, 'init'">
+        <table name="t_order" start-index="12" stop-index="18" />
+        <columns start-index="20" stop-index="46">
+            <column name="order_id" start-index="21" stop-index="28" />
+            <column name="user_id" start-index="31" stop-index="37" />
+            <column name="status" start-index="40" stop-index="45" />
+        </columns>
+        <values>
+            <value>
+                <assignment-value>
+                    <parameter-marker-expression parameter-index="0" start-index="56" stop-index="56" />
+                    <literal-expression value="1" start-index="56" stop-index="56" />
+                </assignment-value>
+                <assignment-value>
+                    <parameter-marker-expression parameter-index="1" start-index="59" stop-index="59" />
+                    <literal-expression value="-1" start-index="59" stop-index="60" />
+                </assignment-value>
+                <assignment-value>
+                    <parameter-marker-expression parameter-index="2" start-index="62" stop-index="62" />
+                    <literal-expression value="init" start-index="63" stop-index="68" />
+                </assignment-value>
+            </value>
+        </values>
     </insert>
 </sql-parser-test-cases>

--- a/shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/dml/insert.xml
+++ b/shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/dml/insert.xml
@@ -71,7 +71,7 @@
     <sql-case id="insert_without_columns_with_default_values" value="INSERT INTO t_order DEFAULT VALUES" db-types="SQLServer" />
     <sql-case id="insert_with_top" value="INSERT TOP(10) t_order (order_id, user_id) SELECT order_id, user_id FROM t_order" db-types="SQLServer" />
     <sql-case id="insert_with_top_percent" value="INSERT TOP(10) PERCENT t_order (order_id, user_id) SELECT order_id, user_id FROM t_order" db-types="SQLServer" />
-    <sql-case id="insert_with_columnname_uuid" value="insert into t_order (id,uuid) values (?, ?)" db-types="PostgreSQL,openGauss" />
+    <sql-case id="insert_with_uuid_column" value="insert into t_order (id,uuid) values (?, ?)" db-types="PostgreSQL,openGauss" />
     <sql-case id="insert_with_output_clause" value="INSERT INTO t_order (order_id, user_id) OUTPUT INSERTED.order_id, INSERTED.user_id INTO @MyTableVar (temp_order_id, temp_user_id) VALUES (?, ?)" db-types="SQLServer" />
     <sql-case id="insert_with_output_clause_without_output_table_columns" value="INSERT INTO t_order (order_id, user_id) OUTPUT INSERTED.order_id, INSERTED.user_id INTO @MyTableVar VALUES (?, ?)" db-types="SQLServer" />
     <sql-case id="insert_with_output_clause_without_output_table" value="INSERT INTO t_order (order_id, user_id) OUTPUT INSERTED.order_id, INSERTED.user_id VALUES (?, ?)" db-types="SQLServer" />
@@ -86,4 +86,5 @@
     <sql-case id="insert_with_schema" value="INSERT INTO db1.t_order VALUES (1,2,3)" />
     <sql-case id="insert_on_duplicate_key_update_nothing" value="INSERT INTO t_order (order_id, user_id, status) VALUES (1, 1, 'insert') ON DUPLICATE KEY UPDATE NOTHING" db-types="openGauss" />
     <sql-case id="insert_on_duplicate_key_update_multi_column" value="INSERT INTO t_order (order_id, user_id, status) VALUES (1, 1, 'insert') ON DUPLICATE KEY UPDATE user_id = user_id + 1, status='update'" db-types="MySQL,openGauss" />
+    <sql-case id="insert_with_negative_value" value="insert into t_order (order_id, user_id, status) values (?, ?, ?)" />
 </sql-cases>

--- a/shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/unsupported/unsupported.xml
+++ b/shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/unsupported/unsupported.xml
@@ -3816,7 +3816,6 @@
     <sql-case id="explain_by_postgresql_source_test_case27" value="EXPLAIN (COSTS OFF) EXECUTE role_inval;" db-types="PostgreSQL" />
     <sql-case id="explain_by_postgresql_source_test_case28" value="EXPLAIN (COSTS OFF) EXECUTE role_inval;" db-types="PostgreSQL" />
     <sql-case id="explain_by_postgresql_source_test_case29" value="EXPLAIN (COSTS OFF) INSERT INTO t2 (SELECT * FROM t1);" db-types="PostgreSQL" />
-    <sql-case id="explain_by_postgresql_source_test_case44" value="EXPLAIN (COSTS OFF) SELECT * FROM circle_tbl WHERE f1 &amp;&amp; circle(point(1,-2), 1)     ORDER BY area(f1);" db-types="PostgreSQL" />
     <sql-case id="explain_by_postgresql_source_test_case45" value="EXPLAIN (COSTS OFF) SELECT * FROM fast_emp4000     WHERE home_base &lt;@ &apos;(200,200),(2000,1000)&apos;::box     ORDER BY (home_base[0])[0];" db-types="PostgreSQL" />
     <sql-case id="explain_by_postgresql_source_test_case46" value="EXPLAIN (COSTS OFF) SELECT * FROM fast_emp4000     WHERE home_base &lt;@ &apos;(200,200),(2000,1000)&apos;::box     ORDER BY (home_base[0])[0];" db-types="PostgreSQL" />
     <sql-case id="explain_by_postgresql_source_test_case48" value="EXPLAIN (COSTS OFF) SELECT * FROM inet_tbl WHERE &apos;192.168.1.0/24&apos;::cidr &gt;&gt; i;" db-types="PostgreSQL" />
@@ -4191,8 +4190,6 @@
     <sql-case id="select_by_postgresql_source_test_case226" value="SELECT * FROM (VALUES (1),(2),(3)) v(r), ROWS FROM( rngfunc_sql(11,11), rngfunc_mat(10+r,13) );" db-types="PostgreSQL" />
     <sql-case id="select_by_postgresql_source_test_case233" value="SELECT * FROM ROWS FROM(generate_series(10,11), get_users()) WITH ORDINALITY;" db-types="PostgreSQL" />
     <sql-case id="select_by_postgresql_source_test_case234" value="SELECT * FROM ROWS FROM(get_users(), generate_series(10,11)) WITH ORDINALITY;" db-types="PostgreSQL" />
-    <sql-case id="select_by_postgresql_source_test_case244" value="SELECT * FROM circle_tbl WHERE f1 &amp;&amp; circle(point(1,-2), 1)     ORDER BY area(f1);" db-types="PostgreSQL" />
-    <sql-case id="select_by_postgresql_source_test_case245" value="SELECT * FROM circle_tbl WHERE f1 &amp;&amp; circle(point(1,-2), 1)     ORDER BY area(f1);" db-types="PostgreSQL" />
     <sql-case id="select_by_postgresql_source_test_case246" value="SELECT * FROM fast_emp4000     WHERE home_base &lt;@ &apos;(200,200),(2000,1000)&apos;::box     ORDER BY (home_base[0])[0];" db-types="PostgreSQL" />
     <sql-case id="select_by_postgresql_source_test_case247" value="SELECT * FROM fast_emp4000     WHERE home_base &lt;@ &apos;(200,200),(2000,1000)&apos;::box     ORDER BY (home_base[0])[0];" db-types="PostgreSQL" />
     <sql-case id="select_by_postgresql_source_test_case248" value="SELECT * FROM fast_emp4000     WHERE home_base &lt;@ &apos;(200,200),(2000,1000)&apos;::box     ORDER BY (home_base[0])[0];" db-types="PostgreSQL" />

